index.html

```diff
@@ -416,65 +416,172 @@ <h2 id="liaisons">
       Specification(s) to be developed in this group can be completed.
       </li>
     </ul>
+
+    <p class="note">Proposal used Community and Business Group Process and Patent Policy as header</p>
+
     <h2 id="process">
-      Community and Business Group Process and Patent Policy
+      Community and Business Group Process
     </h2>
-    <p>
-      The group operates under the <a href=
+
+    <p>The group operates under the <a href=
       "https://www.w3.org/community/about/agreements/">Community and Business
       Group Process</a>. Terms in this Charter that conflict with those of the
       Community and Business Group Process are void.
     </p>
-    <p>
-      As with other Community Groups, W3C seeks organizational licensing
+
+    <p class="note">Proposed charter added sentence from Web Platform Incubator
```

Comment:
Is this an issue with the existing charter?  Can you remove it and file it as a GitHub issue instead?

https://github.com/webscreens/cg-charter/issues